README.md

```diff
@@ -39,7 +39,26 @@ We use images based on a Kolla build, but we've dropped their Ansible approach t
 favour of Helm and native Kubernetes specs. We also have a number of vendor and self developed extensions, especially
 for Neutron, which are build into our images (you can see reference to many of these in the configuration files/values).
 
+## Installation
 
+### Prerequisites
+
+**Kubernetes with Helm**
+
+See https://github.com/kubernetes/kubernetes and https://github.com/kubernetes/helm
+
+**Local Charts server**
+
+During installation, openstack-helm expects a chart server running on localhost. You can start one with the following command:
+
+    helm serve --address localhost:8879
```

Comment:
I think the `--address localhost:8879` is redundant as its the default behaviour of `helm serve`